### PR TITLE
UBO draw instanced sample

### PR DESCRIPTION
**Pull request by trungtle**
As noted in #14, this PR contains only one of the 2 samples that Shrek built. This is the draw_instanced_ubo sample to minimally demonstrate ubo. 

The other sample involves some lighting and specular calculation, is more of a direct port from Christophe's but it still have some issues so I took it out from this PR.

### CODE REVIEW

**Comment by pjcozzi** [comment]
> The other sample involves some lighting and specular calculation, is more of a direct port from Christophe's but it still have some issues so I took it out from this PR.

Did you move this to another branch for now?

File: samples/draw_instanced_ubo.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        precision highp int;
+
+        layout(std140, column_major) uniform;
+
+        uniform Transform
+        {
+            mat4 MVP[2];
+        } transform;
+
+        layout(location=0) in vec2 pos;
```
Comment (pjcozzi) [inline samples/draw_instanced_ubo.html]: We should update this to be like the others.

@shrekshao are there any other consistency tweaks needed in this pull request?

**Comment by trungtle** [comment]
@pjcozzi The other sample is in branch `dev-shrek-buffer-uniform` still

**Comment by shrekshao** [comment]
Oh the layout locations i think. Thanks for pointing this out.

**Comment by trungtle** [comment]
I updated the layout location

**Pull request merged**